test/polynomial.jl

```diff
@@ -51,6 +51,10 @@ const MP = MultivariatePolynomials
     @test coefficient(2x + 4y^2 + 3, y^2) == 4
     @test coefficient(2x + 4y^2 + 3, x^2) == 0
 
+    Mod.@polyvar a b
+    @test coefficient((2a + b)x^2 + 2y^2 + 3, x^2, [x,y]) == 2a + b
```

Comment:
Use `(x, y)` instead of `[x, y]` for it to work with TypedPolynomials

Reply:
Thanks!